framework/src/modules/dpos_v2/module.ts

```diff
@@ -336,6 +340,10 @@ export class DPoSModule extends BaseModule {
 				isBanned: dposValidator.isBanned,
 				pomHeights: dposValidator.pomHeights,
 				consecutiveMissedBlocks: dposValidator.consecutiveMissedBlocks,
+				// TODO: Issue: #7669
+				commission: 0,
+				lastCommissionIncreaseHeight: 0,
+				sharingCoefficients: [{ tokenID: Buffer.alloc(8), coefficient: Buffer.alloc(24) }],
```

Comment:
For the length of the buffers, instead of using number literals, it is much better practice to use constants `TOKEN_ID_LENGTH` and `MAX_NUM_BYTES_Q96` which you have already defined in constants anyway.

Not only here, but in all other parts of the code where you assign values to `sharingCoefficients` array.

Reply:
This is temporary as the issues referred will define intentional values.